src/section/section_authority_provider.rs

```diff
@@ -25,11 +26,13 @@ pub struct SectionAuthorityProvider {
     pub elders: BTreeMap<XorName, SocketAddr>,
     /// The section prefix. It matches all the members' names.
     pub prefix: Prefix,
+    /// Public key set of the section.
+    pub pk_set: Option<ReplicaPublicKeySet>,
```

Comment:
I'm not sure we should use `sn_data_types::ReplicaPublicKeySet` for the pk-set, I see they are the same but sn_data_types::ReplicaPublicKeySet is meant as a type for the transfers types and messages, so perhaps we should use directly the `threshold_crypto::PublicKeySet` ?

Reply:
This should not be a PublicKey set at all but the SectionKey only. So BLS::PublicKey